test_runner/src/main/kotlin/ftl/android/AndroidCatalog.kt

```diff
@@ -65,12 +65,22 @@ object AndroidCatalog {
         return SupportedDeviceConfig
     }
 
-    fun isVirtualDevice(device: AndroidDevice?, projectId: String): Boolean = device?.androidModelId?.let { isVirtualDevice(it, projectId) } ?: false
+    fun isVirtualDevice(device: AndroidDevice?, projectId: String): Boolean = device
+        ?.androidModelId
+        ?.let { isVirtualDevice(it, projectId) }
+        ?: false
 
     fun isVirtualDevice(modelId: String, projectId: String): Boolean {
-        val form = deviceCatalog(projectId).models.find { it.id == modelId }?.form ?: "PHYSICAL"
-        return form == "VIRTUAL"
+        val form = deviceCatalog(projectId).models
+            .find { it.id.equals(modelId, ignoreCase = true) }?.form
+            ?: "PHYSICAL".also { println("Unable to find device type for $modelId. PHYSICAL used as fallback in cost calculations") }
```

Comment:
Use the DeviceType enum that was created below. `DeviceType.Physical.GetName` should do.

Reply:
There are a few places around this that the enum can be used. I wont single out everyone as it is spam. But i think you get the idea 👍

Reply:
👍  Good idea, I don't like depending on strings